README.md

```diff
@@ -98,6 +102,41 @@ export default class Ember.Component.extend({
 }) { }
 ```
 
+### ES6 Compatible
+
+`ember-strong-attr` exposes one function to declare strong attributes on
+`Ember.Component`
+
+- `declareStrongAttrs(attrs, component)`, it returns the modified `component` that was passed in.
+
+Example:
+
+```js
+import Ember from 'ember';
+import { declareStrongAttrs } from 'ember-strong-attrs';
+import Person from '../models/person';
+
+export default declareStrongAttrs({
+  'myRequiredAttr': {
+    type: String,
+    required: true
+  },
+  'myOptionalAttr': {
+    type: String,
+    required: false
+  },
+  'myOtherOptionalAttr': {
+    type: String
+  }
+  'myPersonAttr': {
+    type: Person,
+    required: true,
+  }
+}, Ember.Component.extend({
+  // ... your methods and props
+}));
+```
```

Comment:
This is pretty nice. It gives me another idea, which is to provide a little DSL here, which could make the decorator and non-decorator APIs more similar, and perhaps provide some more expressiveness. Here's what I'm imagining:

```js
export default declareStrongAttrs(function(){
  this.requiredAttr('myRequiredAttr', String);
  this.optionalAttr('myOptionalAttr', String);
  this.optionalAttr('myOtherOptionalAttr', String);
  this.requiredAttr('myPersonAttr', Person);
}, Ember.Component.extend({
  // ... your methods and props
}));
```

Thoughts?